Dockerfile

```diff
@@ -1,22 +1,44 @@
 ARG POSTGRESQL_VERSION="16"
-FROM bitnami/postgresql:$POSTGRESQL_VERSION
+FROM bitnami/postgresql:$POSTGRESQL_VERSION AS builder
 LABEL org.opencontainers.image.source="https://github.com/dbsystel/postgresql-partman-container"
+ARG JOBMON_VERSION="v1.4.1"
```

Comment:
We should move all ARGs in front of the first from to keep them DRY.

Reply:
Unfortunately, ARGs are only valid for a single stage

Sources:
https://stackoverflow.com/a/53682110
https://docs.docker.com/reference/dockerfile/#scope

Reply:
No worries. I participate in OpenSource to learn from each other :-)